src/translations/en.json

```diff
@@ -502,6 +502,7 @@
     "common": {
       "loading": "Loading",
       "cancel": "Cancel",
+      "delete_view": "Delete View",
```

Comment:
https://github.com/home-assistant/home-assistant-polymer/blob/59989a902ce1136cdcb3afa6fb303bb059f2eb05/src/translations/en.json#L1386

Reply:
This should not be in `common` but in the section of lovelace and the views, there actually already is a key for `delete` there.

Reply:
@bramkragten I didn't realize those existed in en.json.